### PR TITLE
Move Swisstopo and Asitvd sources to ngeo

### PR DESCRIPTION
**Pull request by gberaudo**
These sources are not specific to gmf.
They may be useful in many ngeo applications.
In addition, add support for EPSG:2056 projection.

### CODE REVIEW

**Comment by sbrunner** [comment]
=> Fix the example https://travis-ci.org/camptocamp/ngeo/jobs/173954750#L2827

**Comment by gberaudo** [comment]
Done, @sbrunner, it is now ready for review.

File: examples/asitvd.js [hunk 1]
```diff
@@ -12,7 +13,7 @@ var app = {};
 
 
 /** @type {!angular.Module} **/
-app.module = angular.module('app', ['gmf']);
+app.module = angular.module('app', [ngeo.module.name]);
```
Comment (sbrunner) [inline examples/asitvd.js]: don't seam to work ...

**Comment by gberaudo** [comment]
OK, it is all green now. @sbrunner is it good to merge now?

**Comment by gberaudo** [comment]
Thanks for the review.

**Pull request merged**